tests/kphp_ci_pipeline_runner.py

```diff
@@ -232,6 +233,14 @@ def parse_args():
 
     return parser.parse_args()
 
+def _calculate_pytest_jobs_count(percent: int = 5) -> int:
```

Comment:
This naming looks confusing. As far as I can see, `percent` defines part of cores which will NOT be used for pytest jobs.

Reply:
Yes, I agree, I made a reverse percentage by mistake :(